code/modules/mob/mob_helpers.dm

```diff
@@ -464,3 +464,7 @@ GLOBAL_LIST_INIT(organ_rel_size, list(
 /mob/proc/set_skills(datum/skills/new_skillset)
 	skills = new_skillset
 	SEND_SIGNAL(src, COMSIG_MOB_SKILLS_CHANGED, skills)
+
+///Returns the slowdown applied to the mob when moving through liquids like water
```

Comment:
Do these need to be a whole proc instead of a var on mob?

Reply:
If you can convince tivi that adding a mob var is fine, then sure.